fuzz/oss-fuzz-build.sh

```diff
@@ -35,12 +35,27 @@ build_teleport_fuzzers() {
   compile_native_go_fuzzer $TELEPORT_PREFIX/lib/services \
     FuzzParseRefs fuzz_parse_refs
 
+  compile_native_go_fuzzer $TELEPORT_PREFIX/lib/srv/db/cassandra/protocol \
```

Comment:
I wonder if we could have an automated check here. It is easy to forget/be unaware that this file needs to be updated.

Reply:
I had similar thoughts, but don't know of anything easy.  Let me know if you think of anything.  In any case I agree.  In the long term I would like to get go fuzzing to be source of truth for oss-fuzz.  Not just for test cases, but also seed corpus (currently we provide no seeds to oss-fuzz).